zerver/tornado/event_queue.py

```diff
@@ -618,7 +625,11 @@ def load_event_queues(port: int) -> None:
             )
 
     for client in clients.values():
-        # Put code for migrations due to event queue data format changes here
+        # The following block will be removed once
+        # we have cleared out all old event queues.
+        if client.realm_host == "":
+            realm = get_realm_by_id(client.realm_id)
+            client.realm_host = realm.host
```

Comment:
Added this by taking 69e5e2c3d as a reference.

Reply:
This is a one-time migration, so in some sense very cheap, but it might involve significant startup delay on a busy server to block on all these queries before processing events again. 

I guess it's worth thinking about whether we really need this -- could we just send `realm_host` from Django to Tornado in the one event type that needs this?